ros2_ws/Dockerfile

```
@@ -138,3 +138,12 @@ COPY --from=ros2-control / /
 
 # start as ROS user
 USER ${USER}
+
+# Metadata
+ARG VERSION
+ARG BASE_TAG
+LABEL org.opencontainers.image.title="AICA ROS2 image"
+LABEL org.opencontainers.image.description="AICA base ROS2 image (includes ros2_control)"
```

nitpick: follow ROS 2 branding guidelines, since this is a base public image it would be nice. In other place (code, comments, etc) the informal ros2 is acceptable. 
```suggestion
LABEL org.opencontainers.image.title="AICA ROS 2 image"
LABEL org.opencontainers.image.description="AICA base ROS 2 image (includes ros2_control)"
```

---

I'll do it in #102